--- CMakeLists.txt	2015-04-28 22:59:15.000000000 +0800
+++ CMakeLists.txt	2018-10-01 23:30:48.236628600 +0800
@@ -44,6 +44,8 @@
 	set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY_DEBUG ${CMAKE_BINARY_DIR}/lib)
 	set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY_RELEASE ${CMAKE_BINARY_DIR}/lib)
 endif(CMAKE_CL_64)
+
+add_definitions(-DUNICODE)
 	
 # Precompiled headers stuff (CMake doesn't have a standard command for enabling precompiled headers,
 # so we have to use a macro)
@@ -96,9 +98,9 @@
 
 # Other CMakeLists are located in project subdirectories 
 
-add_subdirectory("demos/ConsoleDemo")
-add_subdirectory("demos/WTLDemo")
-add_subdirectory("demos/MFCDemo")
+#add_subdirectory("demos/ConsoleDemo")
+#add_subdirectory("demos/WTLDemo")
+#add_subdirectory("demos/MFCDemo")
 
 add_subdirectory("reporting/crashrpt")
 add_subdirectory("reporting/crashsender")
@@ -106,7 +108,7 @@
 add_subdirectory("processing/crashrptprobe")
 add_subdirectory("processing/crprober")
 
-add_subdirectory("tests")
+#add_subdirectory("tests")
 
 # Set output directory for LIB files
 if(CMAKE_CL_64)
