--- a/src/DictConverter.hpp
+++ b/src/DictConverter.hpp
@@ -25,6 +25,6 @@
 * Converts a dictionary from a format to another.
 * @ingroup opencc_cpp_api
 */
-void ConvertDictionary(const string inputFileName, const string outputFileName,
+OPENCC_EXPORT void ConvertDictionary(const string inputFileName, const string outputFileName,
                        const string formatFrom, const string formatTo);
 }
