diff --git a/src/xlsx/xlsxzipreader.cpp b/src/xlsx/xlsxzipreader.cpp
--- a/src/xlsx/xlsxzipreader.cpp
+++ b/src/xlsx/xlsxzipreader.cpp
@@ -48,7 +48,7 @@
 
 void ZipReader::init()
 {
-    QList<QZipReader::FileInfo> allFiles = m_reader->fileInfoList();
+    auto allFiles = m_reader->fileInfoList();
     foreach (const QZipReader::FileInfo &fi, allFiles) {
         if (fi.isFile)
             m_filePaths.append(fi.filePath);
